sdk/storage/azure-storage-files-shares/src/share_client.cpp

```diff
@@ -224,14 +224,17 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
         m_shareUrl, *m_pipeline, context, protocolLayerOptions);
   }
 
-  Azure::Response<Models::GetShareAccessPolicyResult> ShareClient::GetAccessPolicy(
+  Azure::Response<std::vector<Models::SignedIdentifier>> ShareClient::GetAccessPolicy(
       const GetShareAccessPolicyOptions& options,
       const Azure::Core::Context& context) const
   {
     (void)options;
     auto protocolLayerOptions = Details::ShareRestClient::Share::GetAccessPolicyOptions();
-    return Details::ShareRestClient::Share::GetAccessPolicy(
+    auto result = Details::ShareRestClient::Share::GetAccessPolicy(
```

Comment:
we'd better move original return types like `ShareGetAccessPolicyResult` to `Details` namespace

Reply:
It's already in `_detail` namespace.